src/ui/src/components/blueprints/BlueprintsAutogen.vue

```diff
@@ -1,25 +1,46 @@
 <template>
 	<div class="BlueprintsAutogen">
+		<h2>
+			{{
+				isBusy
+					? "Generating agent"
+					: "What would you like this agent to do?"
+			}}
+		</h2>
+		<BlueprintsAutogenContents :class="{ busy: isBusy }">
+			<template v-if="!isBusy">
+				<div
+					class="BlueprintsAutogen__main"
+					:class="{ error: errorMessage }"
+				>
+					<WdsTextareaInput
+						v-model="prompt"
+						rows="10"
+						placeholder="Describe the blocks you'd like to generate..."
+						style="height: 100%"
+					></WdsTextareaInput>
+				</div>
+			</template>
+
+			<template v-else>
+				<BlueprintsGenerationLoader></BlueprintsGenerationLoader>
+			</template>
+		</BlueprintsAutogenContents>
 		<template v-if="!isBusy">
-			<div class="main">
-				<WdsTextareaInput
-					v-model="prompt"
-					rows="8"
-					placeholder="Describe the blocks you'd like to generate..."
-				></WdsTextareaInput>
+			<div class="BlueprintsAutogen__panel">
+				<div class="BlueprintsAutogen__error">
+					{{ errorMessage }}
+				</div>
```

Comment:
1. It's better to use `<p>` instead of `<div>`
2. you can hide if there are no messages
```suggestion
				<p v-if="errorMessage" class="BlueprintsAutogen__error">
					{{ errorMessage }}
				</p>
```

Reply:
I switched to `<p>` but didn't add `v-if` because that would break positioning of the buttons. I have `justify-content: space-between;` on the `BlueprintsAutogen__panel`. So non-existence of the `BlueprintsAutogen__error` forces buttons to the left side